### PR TITLE
Welding Mask Fix

### PR DESCRIPTION
**Pull request by Geevies**
* You can now flip your welding mask up and down even when buckled.

### CODE REVIEW

**Comment by Geevies** [comment]
!bugfix

**Comment by Geevies** [comment]
!review

File: code/modules/clothing/head/misc_special.dm [hunk 1]
```diff
@@ -51,27 +51,33 @@
...
-			item_state = "[base_state]up"
-			to_chat(usr, "You push the [src] up out of your face.")
-		update_clothing_icon()	//so our mob-overlays
-		usr.update_action_buttons()
+	if(usr.stat)
+		to_chat(usr, span("warning", "You can't do that in this state!"))
+		return
+	if(usr.restrained())
+		to_chat(usr, span("warning", "You can't reach your [src]!"))
+		return
```
Comment (NonQueueingMatt) [inline code/modules/clothing/head/misc_special.dm]: use_check_and_message

File: code/modules/clothing/head/misc_special.dm [hunk 2]
```diff
@@ -51,27 +51,33 @@
...
+
+	if(src.up)
+		src.up = !src.up
```
Comment (NonQueueingMatt) [inline code/modules/clothing/head/misc_special.dm]: This and the src.up below are duplicates.

**Pull request merged**